applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/bt_rendering_and_capture.c

```diff
@@ -102,13 +126,12 @@ int bt_rend_volume_unmute(void)
 	return ret;
 }
 
-int bt_rend_discover(struct bt_conn *conn)
+int bt_r_and_c_discover(struct bt_conn *conn)
 {
 	int ret;
 
-	/* Only do a VCS discover if we are volume controller */
 	if (IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
```

Comment:
Should there be an else here and a warning print that Discover is not supported?

Reply:
Added some error returns.